### PR TITLE
Add issue template for bug report

### PR DESCRIPTION
**Pull request by loichuder**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior** A clear and concise description of what you expected to
+happen.
+
+**Screenshots** If applicable, add screenshots to help explain your problem.
+
+**Context (please complete the following information):**
```
Comment (axelboc) [inline .github/ISSUE_TEMPLATE/bug_report.md:23]: maybe just `**Context**` -- if you put any mention of "required" or similar, people might think the other information are optional. ^^

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+- OS: [e.g. iOS]
+- Browser: [e.g. chrome, safari]
+- Version: [e.g. 22]
+- h5web context: [choose one or several] jupyterlab_h5web, @h5web/app,
+  @h5web/lib, demo ?
```
Comment (axelboc) [inline .github/ISSUE_TEMPLATE/bug_report.md:29]: I would reorder this: `demo, jupyterlab_h5web, @h5web/lib, @h5web/app`

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+---
+
+**Describe the bug** A clear and concise description of what the bug is.
```
Comment (axelboc) [inline .github/ISSUE_TEMPLATE/bug_report.md:9]: Maybe use `h4` for headings with `####`? The descriptions could go on their own lines, and be wrapped in comments so they don't appear if people leave them in:

```md
#### Describe the bug

<!-- A clear and concise description of what the bug is -->
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
```
Comment (axelboc) [inline .github/ISSUE_TEMPLATE/bug_report.md:6]: Do these empty fields have to be specified?

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+---
+
+**Describe the bug** A clear and concise description of what the bug is.
+
+**To Reproduce** Steps to reproduce the behavior:
+
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
```
Comment (axelboc) [inline .github/ISSUE_TEMPLATE/bug_report.md:16]: I'd include this in the comment with the description.

**Review by axelboc — APPROVED** [review]
Excellent idea 💯

**Comment by loichuder** [comment]
Thanks for the comments, I reused most of Github default template without paying too much attention

**Pull request merged**